backend/lib/components/Geo/gmaps/index.js

```diff
@@ -50,8 +128,39 @@ const getLocationByLatLng = async (lat, lng) => {
   };
 };
 
+const getHealthFacilityPlaces = async (lat, lng, type) => {
+  const MAX_FACILITIES = 10;
+  const location = { lat: lat, lng: lng };
+  const radius = FACILITY_RADIUS.find(o => o.type === type).radius;
+  
+  let places = [];
+  
+  await getNearbyPlacesByType({ key: geo.googleMapsApiKey, location: location, type: type, radius: radius }, MAX_FACILITIES )
+    .then((result) => {
+      result.forEach(function(item) {
+        const place = new Place(item.name, item.place_id, type); 
+        places.push(place);
```

Comment:
@maskowe do we need this `Place` class? Looking at the rest of the code it doesn't seem like this functionality is not really used. 

The only difference I see in the output after removing this is lack of `type` with the response, which could be achieved in a much more simple way.

Reply:
Thanks.  I included the type originally in the class because originally the client was not sending that in.   Even so, I think it's more convenient, but if you don't like it, I can certainly take it out.  Also the class includes distance, which extends the rest of the place details, being part of the different api.   I have pretty much built into the place class, since I'm doing the FE stuff and will know the structure based on this class.   But if everything comes back that I need (everything in the place instance), then I'm good with however you want to work that.

Reply:
Some of the functionality is not used (getters and setters), because I thought maybe unit testing could use those...Again, can take it out if you want.  It's ok for you to work it however you want, just let me know and thanks again.

Reply:
>  Also the class includes distance, which extends the rest of the place details, being part of the different api.

Aren't you getting distance here? 

https://github.com/FightPandemics/FightPandemics/blob/b5bf662f4484a007e8589b02ad8a23410345a5cd/backend/lib/components/Geo/gmaps/index.js#L71

Reply:
Yes - distance: data.rows[0].elements[i].distance.text  provides the distance by index position to the place instance property called "distance".   That's what I meant when I said "extends" the class..